packages/flame/pubspec.yaml

```diff
@@ -7,13 +7,13 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.3.0
-  ordered_set: ^3.1.0
+  ordered_set: ^3.2.0
   vector_math: '>=2.1.0 <3.0.0'
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  test: ^1.16.0
+  test: ^1.17.10
```

Comment:
I wonder if we need to keep depending on this, we already get test methods on the `flutter_test` dependency.

Either way this is unrelated to your PR, so no need to remove it on this PR.

Reply:
good q, I imagine it might be bc of this import `package:test/fake.dart`? def worth testing out though

Reply:
Makes sense, we probably still needs it then.